core/src/i18n/en.yaml

```
@@ -469,6 +474,15 @@ deployments:
   title: Deployments
 discovery-and-network:
   title: Discovery and Network
+domains:
+  exclude:
+    label: Exclude Domains
+    placeholder: Domain that should be ignored
```

"should" - is this a recommendation? Or is it certainly going to be ignored if entered here?
Based on the following line, how about

 ```suggestion
    placeholder: Domain that is skipped for certificate requests
```

---

Not sure if `is skipped` is a good explanation here. This placeholder should be a short version of this:
```The domains (including subdomains) for which certificate requests are forbidden, for example, *.api.mydomain.com.```

---

That's what I was confused about - at one place we write "ignored", somewhere else "excluded", and then also "forbidden".  They're all related but don't mean the same thing. 
My assumption was that it's basically an allowlist of domains that are always "good" and certificate never must be checked. As opposed to the Include domains that always must be checked. Is that correct?